indexer/processors/contracts_bedrock/utils.go

```diff
@@ -0,0 +1,63 @@
+package contracts_bedrock
```

Comment:
Is there a better place this code could live later? This is golang sdk utils
```suggestion
// Consider moving me to a general utils package for bedrock
package contracts_bedrock
```

Reply:
yea we probably move the nonce decoding logic elsewhere